src/ertlibc/customfs.c

```diff
@@ -172,6 +208,18 @@ static int _fs_mount(
     if ((flags & OE_MS_RDONLY))
         fs->mount.flags = flags;
 
+    /* ---------------------------------------------------------------------
+     * Only support absolute paths. Hostfs is treated as an external
+     * filesystem. As such, it does not make sense to resolve relative paths
+     * using the enclave's current working directory.
+     * ---------------------------------------------------------------------
+     */
+    if (source && source[0] != '/')
+        OE_RAISE_ERRNO(OE_EINVAL);
+
+    /* Save the source parameter (will be needed to form host paths). */
+    oe_strlcpy(fs->mount.source, source, sizeof(fs->mount.source));
```

Comment:
I am not sure how mounting exactly works behind the scenes in Open Enclave, but can we maybe also resolve the correct source path based on existing mount points?

For my PR in https://github.com/edgelesssys/ego/pull/49, I have to treat the source path differently even though the memfs is mounted at /edg/mnt. 

I think it would help to clear up some confusion, but if it's too complicated or has other issues I think we can leave it how it is.

Reply:
I guess we'd need to change code in OE for that, so I'd rather leave it as is.

Reply:
Okay. It's not quite ideal, but guess if the alternative is to make changes to OE's mount code, we should just leave it I guess and keep this here as some kind of documentation :)